cmd/dep/status.go

```diff
@@ -81,6 +82,13 @@ type outputter interface {
 	MissingFooter() error
 }
 
+// Only a subset of the outputters should be able to output old statuses
```

Comment:
Let's end all the comments with a period (`.`). We have been doing that for new code comments :)